### PR TITLE
[MKC-1854] GIGA CAN / ADC fixes

### PR DESCRIPTION
**Pull request by karlsoderby**
The GIGA documentation had incorrectly labelled CANTX/CANRX pins (D93/D94) as ADC pins. These pins are not connected to the ADC.

- Update CANTX / CANRX mentions in CS / Audio guide.

## Contribution Guidelines
- [x] I confirm that I have read the [contribution guidelines](https://github.com/arduino/docs-content/tree/main/contribution-templates) and comply with them.

### CODE REVIEW

**Review by Hannes7eicher — CHANGES_REQUESTED** [review]
I think the auto formatter got a little too excited and broke a few note banners :))

Although it also includes many fixes I would suggest we only change the pins and if we want to clean up articles we make a seperate PR. 

What do you think?

**Comment by karlsoderby** [comment]
@Hannes7eicher totally agree. Changes fixed!

File: content/hardware/10.mega/boards/giga-r1-wifi/tutorials/cheat-sheet/cheat-sheet.md [hunk 1]
```diff
@@ -1001,106 +1001,111 @@ Serial1.write("Hello world!");
...
+| A12 / D84 | DAC0            | Analog In, DAC                          |
+| A13 / D85 | DAC1            | Analog In, DAC                          |
+| D86       | RGB (red)       | Only RGB, not accessible as GPIO        |
+| D87       | RGB (green)     | Only RGB, not accessible as GPIO        |
+| D88       | RGB (blue)      | Only RGB, not accessible as GPIO        |
+| D89       | SPI1 (CIPO)     | SPI connector                           |
+| D90       | SPI1 (COPI)     | SPI connector                           |
+| D91       | SPI1 (SCK)      | SPI connector                           |
+| D92       | USB Host Enable | USB-A connector, not accessible as GPIO |
+| D93       | CANRX           | Analog In, CAN                          |
```
Comment (Hannes7eicher) [inline content/hardware/10.mega/boards/giga-r1-wifi/tutorials/cheat-sheet/cheat-sheet.md:1107]:
```suggestion
| D93       | CANRX           | Digital IO pin, CAN                     |
```

File: content/hardware/10.mega/boards/giga-r1-wifi/tutorials/cheat-sheet/cheat-sheet.md [hunk 2]
```diff
@@ -1001,106 +1001,111 @@ Serial1.write("Hello world!");
...
+| D94       | CANTX           | Analog In, CAN                          |
```
Comment (Hannes7eicher) [inline content/hardware/10.mega/boards/giga-r1-wifi/tutorials/cheat-sheet/cheat-sheet.md:1108]:
```suggestion
| D94       | CANTX           | Digital IO pin, CAN                     |
```

**Review by Hannes7eicher — APPROVED** [review]
two super small fixes I missed during the last review, otherwisee looks good!

**Pull request merged**